<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UniPlanner</class>
 <widget class="QMainWindow" name="UniPlanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Uniplanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Banner">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>601</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(139, 45, 32, 186);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700; color:#ffffff;&quot;&gt;Uniplanner&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="InformationsOverskrift">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 209);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Velkommen til dit universitets personlige skema- og booking system&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Bruger">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Bruger: &lt;/span&gt;Underviser / Lisbeth Lund / Københavns Universitet / Humanbiologi&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="MenuBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 170);
color: rgb(255, 255, 255);
border-style:outset;
border-radius:10px;
font: 13pt &quot;Arial&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>Menu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Søg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kurser</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Studerende</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Underviser</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="UniSkema">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>521</width>
      <height>351</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>521</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>521</width>
      <height>351</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(139, 45, 32);</string>
    </property>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>16</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>17</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>18</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Mandag</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tirsdag</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Onsdag</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Torsdag</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fredag</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <item row="0" column="1">
     <property name="text">
      <string>Forelæsning: Humanbiologi Aud. Einer Lundsgaard</string>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>Forelæsning: Humanbiologi Aud. Niels K. Jerne</string>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>Forelæsning: Humanbiologi Aud. Einer Lundsgaard</string>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string>Forelæsning: Humanbiologi Aud. Niels K. Jerne</string>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>Forelæsning: Humanbiologi Aud. Einer Lundsgaard</string>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string>Forelæsning: Humanbiologi Aud. Einer Lundsgaard</string>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
    </item>
    <item row="4" column="4">
     <property name="text">
      <string>Forelæsning: Humanbiologi Aud. Einer Lundsgaard</string>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
    </item>
    <item row="5" column="4">
     <property name="text">
      <string>Forelæsning: Humanbiologi Aud. Einer Lundsgaard</string>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
    </item>
    <item row="6" column="4">
     <property name="text">
      <string>Forelæsning: Humanbiologi Aud. Einer Lundsgaard</string>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Fejl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Fejl?&lt;/span&gt; Kontakt adminstrationen (+45 12 34 56 78)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NaesteUge">
    <property name="geometry">
     <rect>
      <x>499</x>
      <y>140</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Næste uge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ForrigeUge">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Forrige uge</string>
    </property>
   </widget>
   <widget class="QLabel" name="Dato">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Uge 11 - Marts - 2022&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LogUd">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(139, 45, 32);
color: rgb(255, 255, 255);
border-style:outset;
border-radius:10px;
font: 13pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Log ud</string>
    </property>
   </widget>
   <widget class="QFrame" name="SkemaFrame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>541</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(170, 170, 170, 129);
border-style:outset;
border-radius:10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Lokale_aendringer_skift">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 170);
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-style:outset;
border-radius:10px;
font: 13pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Anmode om ændringer/skift af lokale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Tilgaengelighed">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 170);
color: rgb(255, 255, 255);
border-style:outset;
border-radius:10px;
font: 13pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Tilgængelighed</string>
    </property>
   </widget>
   <zorder>Lokale_aendringer_skift</zorder>
   <zorder>Tilgaengelighed</zorder>
   <zorder>SkemaFrame</zorder>
   <zorder>Banner</zorder>
   <zorder>InformationsOverskrift</zorder>
   <zorder>Bruger</zorder>
   <zorder>MenuBox</zorder>
   <zorder>Fejl</zorder>
   <zorder>UniSkema</zorder>
   <zorder>NaesteUge</zorder>
   <zorder>ForrigeUge</zorder>
   <zorder>Dato</zorder>
   <zorder>LogUd</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
